golang/btcspv/utils.go

```diff
@@ -14,10 +14,47 @@ func strip0xPrefix(s string) string {
 	return s
 }
 
+// Decodes a hex string into a byte array
 func DecodeIfHex(s string) []byte {
 	res, err := hex.DecodeString(strip0xPrefix(s))
 	if err != nil {
 		return []byte(s)
 	}
 	return res
 }
+
+func GetOutputType(num uint) string {
+	var typeString string
+	switch num {
+	case 0:
```

Comment:
Actually, now that I think about it, we can use the enum names for the cases 

`case OutputNone`

Reply:
Yes, that worked 🙂